src/devices/SoftwareSpi/samples/README.md

```diff
@@ -0,0 +1,12 @@
+﻿# Software implementation of SPI
+
+Usage is very simple:
+
+```csharp
```

Comment:
the READMEs are wrong, this should go under SoftwareSpi/ and this file should only point to Mcp3008 sample and all .cs files in this folder removed